### ISSUE TITLE
Deploy eShop to k8s via Aspir8

### ISSUE THREAD

**Issue by danmoseley** [dotnet/aspire#1964]

**Comment by ranma42** [comment]
Is the eShop supposed (or, even better, known) to work with `aspirate`?
I tried to emit its `compose` manifest, but I hit https://github.com/prom3theu5/aspirational-manifests/issues/262

(note that it is my first attempt at using `aspirate`, so I might simply be mis-using it)

**Comment by John-Leitch** [comment]
`python-inference` is an `executable.v0`, which isn't supported by Aspirate. It doesn't help that this causes a `NullReferenceException` due to references evaluation issues. I plan on fixing Aspirate to address the null deref and adding warnings for unsupported resource types, but that just means nicer error messages. Support for `executable.v0` probably isn't very realistic.

**Comment by ranma42** [comment]
Cool! If that is the issue, maybe a dockerized version of `python-inference` would work (or maybe I'll hit another issue 😅 ).
Thank you for getting back to me and pointing out what was going on.
When I tried the first time, I was unable to understand the error so I just gave up after that attempt, but I will try again soon with a modern version of aspirate (iiuc 9.1.0 would already provide a more readable error 🚀 ).

**Comment by John-Leitch** [comment]
@ranma42, yes, 9.1 includes some error output improvements as well as `container.v1` support, which may be needed if `python-inference` is containerized. I'm not familiar with this project, so not sure what other issues you may hit. Looking at the manifest a little more, I see there's an `azure.bicep.v0` as well. You'll probably have to do something about that too.

**Comment by maddymontaquila** [comment]
Should pivot this to be via the Kubernetes publisher
